scripts/rabbitmq-server-ha.ocf

```diff
@@ -1619,6 +1640,45 @@ get_monitor() {
     return $rc
 }
 
+order_node_restart() {
+    local node=${1:?}
+    local nowtime
+    nowtime=$(now)
+    ocf_log warn "${LH} Ordering node '$node' to restart"
+    ocf_update_private_attr 'rabbit-ordered-to-restart' "$nowtime" "$node"
```

Comment:
Maybe just inline $(now) here? We will save two lines :-) That is just an opinion